### ISSUE TITLE
Adding non-morphological traits

### ISSUE THREAD

**Issue by AnahiEspindola** [phenoscape/TraitFest-2023#13]
From a user perspective, and thinking about important traits in insects, having databasing of ecological traits would be really important. For example, host plants, parasite hosts, range spanning, environmental preferences. In many insect groups these are sometimes used in species delimitation, and having this information entered would be very really important IMO. Some of this is also often present in labels, so I don't think this would be super crazy to add... maybe?

**Comment by seltmann** [comment]
I am also interested in this for bee functional traits such as sociality, floral host, and phenology. At least in part, this would correspond to the issue of alignment with other ontologies such as UBERON & RO https://github.com/phenoscape/TraitFest-2023/issues/10

**Comment by meghalithic** [comment]
There are some ontologies attempting this, like [GLOBI](https://www.globalbioticinteractions.org/) (biotic interactions, potentially parasites), [PPO](https://github.com/PlantPhenoOntology/ppo) (phenology, which includes occurrence data), and [OBA](https://bioportal.bioontology.org/ontologies/OBA) (attributes of species), [FuTRES](https://futres.org/dashboard/) (vertebrate morphological traits), and I think @JCGiron is working on one for insects. Do these touch on what you mean, knowing that no one resource does all the things or focuses on just insects? If so, perhaps they could be brought into Phenoscape!

**Comment by seltmann** [comment]
@megbalk yes for them being brought into Phenoscape. I work with GloBIand we use RO for the relationships. Relates to https://github.com/phenoscape/TraitFest-2023/issues/10

**Comment by JCGiron** [comment]
Yes!
General for insects we have [AISM](https://github.com/insect-morphology/aism), and derived from it [COLAO](https://github.com/insect-morphology/colao) for beetles, and [LEPAO](https://github.com/insect-morphology/lepao) for butterflies and moths.
There is also [HAO](http://portal.hymao.org/projects/32/public/ontology/), totally usable for annotation purposes but still needs to be integrated into the ODK ([Ontology Development Kit](https://github.com/INCATools/ontology-development-kit)) world.

**Comment by megbalk** [comment]
We still want the output of the presence / absence matrix to do our preliminary analyses and set up for the next paper. Can we add that back in?
